meson.build

```diff
@@ -1261,7 +1262,6 @@ endif
 # vulkan
 vulkan_opt = get_option('vulkan').require(
     libplacebo.get_variable('pl_has_vulkan', default_value: '0') == '1',
-    error_message: 'libplacebo could not be found!',
```

Comment:
`error_message: 'libplacebo compiled without Vulkan support!',` or something like that